### PR TITLE
add print interpreter api

### PR DESCRIPTION
**Pull request by zh794390558**

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @jdduke: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by ymodak** [comment]
@jdduke Can you please take a look at this PR? Thanks!

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @jdduke: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @jdduke: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @aselle: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

File: tensorflow/contrib/lite/experimental/c/c_api.h [hunk 1]
```diff
@@ -167,6 +167,10 @@ TFL_CAPI_EXPORT extern int32_t TFL_InterpreterGetOutputTensorCount(
 TFL_CAPI_EXPORT extern const TFL_Tensor* TFL_InterpreterGetOutputTensor(
     const TFL_Interpreter* interpreter, int32_t output_index);
 
+// Prints a dump of what tensors and what nodes are in the interpreter.
```
Comment (jdduke) [inline tensorflow/contrib/lite/experimental/c/c_api.h]: I don't know that we want to put this in the public stable API quite yet, what about moving to c_api_experimental.h?

**Comment by rthadur** [comment]
These files are moved out of contrib folder , closing this PR

**Pull request closed without merging**